### PR TITLE
Fedora hardware acceleration packages

### PR DESCRIPTION
**Pull request by Man2Dev**
Added Fedoras dependencies for OpenCL, and CLBlast hardware acceleration

### CODE REVIEW

**Review by phymbert — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -574,6 +574,12 @@ Building the program with BLAS support may lead to some performance improvements
 
   Pre-built CLBlast binaries may be found on the [CLBlast Releases](https://github.com/CNugteren/CLBlast/releases) page. For Unix variants, it may also be found in your operating system's packages.
 
+  Linux packaging:
```
Comment (phymbert) [inline README.md:577]: I dont think we want to detail all the install command for all distro.

**Review by phymbert — DISMISSED** [review]
No need to detail how to install clblast

**Comment by Man2Dev** [comment]
> No need to detail how to install clblast

the instruction only states how to build it but not that it's avalibale in the native package manager.  which can create pointless work if some one is uninformed

**Comment by phymbert** [comment]
I just worry the README.md on the welcome page will become difficult to read if we add so much detailled instruction

**Comment by Man2Dev** [comment]
> I just worry the README.md on the welcome page will become difficult to read if we add so much detailled instruction

I understand your predicament however, not accepting the patch won't fix the problem the readme is already riddled it inconsistencies and has no sense of cohesion which makes it arguably more un readable. 
The way to actually address this issue is by firstly re formatting the readme to make it more cohesive and much easier to read. And having gidelines for formatting which preferbly be inforced by CI.
However, if you are still against merging  the pr then how about I write all the fedora instructions in different  markdown file and just have the readme link to that? (But then some one has to port the windows, ubuntu ... to there own file at some point as to make it more coherent) in my opinion it would be best to just merge it at this point untill some gidelines have been put in place for documentation.

**Review by phymbert — COMMENTED** [review]
@ggerganov hi

**Review by ggerganov — APPROVED** [review]
The README can be improved, but lets merge for now

**Pull request merged**